ParameterDataApi.yaml

```diff
@@ -143,60 +199,69 @@ components:
     HeaderDefinition:
       type: object
       properties: 
-        startTime:
-          description: The start time for the time series
-          type: string
-          format: date-time
-        
-        endTime: 
-          description: The end time for the time series
+        messageType:
+          type: string          
+          description: The type of the message
+
+        messageVersion:
           type: string
-          format: date-time
+          description: The version string for the parameter data messageType
           
-        columns:
-          $ref: '#/components/schemas/ColumnDefinition'
-        
       required:
-        - startTime
-        - endTime
+        - messageType
+        - version
+
+    SingleDataDefinition:
+      type: object
+      description: A definition of a single parameter data sample. This is loosely based upon Json Time Series, Version 1.0 (https://docs.eagle.io/en/latest/reference/index.html)
+      properties:
+
+        header:
+          $ref: '#/components/schemas/HeaderDefinition'
+          
+        columns:
```

Comment:
If we're going to make a separate endpoint for this entirely, why bother specifying columns for single data samples? They sort of make sense for the batched time series data to strictly enforce homogeneity in the data, but the single sample can just be a collection of:

"fieldName" : {
  "value": \<value\>,
  "type": \<type\>,
  "quality": \<quality\>,
  "annotation": \<annotation\>
}

Reply:
Also, as discussed previously, I still question how useful the quality and annotation fields would be in practice, as there is no obvious need or use case for both single samples and time series. Really feels like this could be as simple as:

"fieldName" : \<value\>

Reply:
I have added units to the ColumnElement. In the previous version, we were using the annotation field for this, but this is better guidance as a dedicated and very important field. 

As for doing away with the ColumnDefinition for single samples, I'm concerned that this would require us to have two different Data Element types, one with units for single sample requests and one without for time series requests.